app/models/spree/ability_decorator.rb

```diff
@@ -155,6 +155,8 @@ def add_product_management_abilities(user)
     can [:admin, :index, :read, :search], Spree::Taxon
     can [:admin, :index, :read, :create, :edit], Spree::Classification
 
+    can [:admin, :index, :import, :save], ProductImporter
```

Comment:
@oeoeaio I think any users that are able to manage products will be able to use the product import, yes. There are checks by the importer on a per-item basis to see if the user has permission to add/edit products for the enterprise they are trying to import into.